cmd/sqlcmd/sqlcmd.go

```diff
@@ -162,11 +162,44 @@ func Execute(version string) {
 		},
 	}
 	setFlags(rootCmd, &args)
+	rootCmd.SetHelpFunc(func(cmd *cobra.Command, argss []string) {
+		fmt.Println(cmd.Long)
```

Comment:
>Println(cmd.Long) [](http://example.com/codeflow?start=6&length=17)

we should print the version too

I'm not certain about the copyright.

eg this is what ODBC sqlcmd has at the top

```
Microsoft (R) SQL Server Command Line Tool
Version 15.0.1300.359 NT
Copyright (C) 2019 Microsoft Corporation. All rights reserved.
```